<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>690</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Sistema Experto en Anime</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="verticalWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>541</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(Menu3.png)</string>
   </property>
   <layout class="QVBoxLayout" stretch="0,0">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1052</width>
        <height>119</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(tr.png)

</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Sin título-2.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="gridFrame_2">
      <property name="styleSheet">
       <string notr="true">background-image: url(Tr.png)</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QFrame" name="gridFrame">
         <property name="maximumSize">
          <size>
           <width>700</width>
           <height>4000</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(Menu4.png);
</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="6" column="0">
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(Tr.png);
color: rgb(255, 255, 255);
background-color: rgb(2, 7, 43);
</string>
            </property>
            <property name="text">
             <string>¿Que genero deseas ver?</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QComboBox" name="Clasificacion">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">selection-background-color: blue;border-width: 2px;
background-image: url(Tr.png);
background-color: white;
border-style: outset;
border-width: 5px;
border-color: White;
font: bold 20px;
min-width: 10em;
padding: 6px;</string>
            </property>
            <item>
             <property name="text">
              <string>Accion</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Romance</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Comedia</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Drama</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(Tr.png);
color: rgb(255, 255, 255);
background-color: rgb(2, 7, 43);

</string>
            </property>
            <property name="text">
             <string>¿Cual es la trama del anime?</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(Tr.png);
color: rgb(255, 255, 255);
background-color: rgb(2, 7, 43);
</string>
            </property>
            <property name="text">
             <string>¿Cual es la edad de la persona?</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(Tr.png);
color: rgb(255, 255, 255);
background-color: rgb(2, 7, 43);
</string>
            </property>
            <property name="text">
             <string>¿Cantidad de capítulos?</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QComboBox" name="Motivo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">selection-background-color: blue;border-width: 2px;
background-image: url(Tr.png);
background-color: white;
border-style: outset;
border-width: 5px;
border-color: White;
font: bold 20px;
min-width: 10em;
padding: 6px;</string>
            </property>
            <property name="currentText">
             <string>Fantasía</string>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
            <item>
             <property name="text">
              <string>Fantasía</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ciencia Ficcion</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Misterio</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Vida Diaria</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="Edad">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">selection-background-color: blue;border-width: 2px;
background-image: url(Tr.png);
border-style: outset;
border-width: 5px;
border-color: White;
font: bold 20px;
min-width: 10em;
padding: 6px;</string>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Niños</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Jovenes</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Adultos</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QComboBox" name="GeneroP">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">selection-background-color: blue;border-width: 2px;
background-image: url(Tr.png);
background-color: white;
border-style: outset;
border-width: 5px;
border-color: White;
font: bold 20px;
min-width: 10em;
padding: 6px;</string>
            </property>
            <item>
             <property name="text">
              <string>30 - 150</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>150+</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>500+</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="widget" native="true">
         <property name="styleSheet">
          <string notr="true">background-image: url(Tr.png);width: 2px;
    border-radius: 25px;</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="2" column="1">
             <widget class="QLabel" name="LabelNombre_2">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-image: url(Tr.png);    border-radius: 5px;color: rgb(255, 152, 56);</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="LabelNombre">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-image: url(Tr.png);    border-radius: 5px;color: rgb(255, 152, 56);</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="labelImagen">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">background-image: url(Tr.png);    border-radius: 5px; weight: 169px; lenght: 178px
</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QLabel" name="label_Explicacion2">
              <property name="maximumSize">
               <size>
                <width>600</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>WhatsThisCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QLabel" name="labelImagen_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">background-image: url(Tr.png);    border-radius: 5px; weight: 169px; lenght: 178px
</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="Label_Explicacion">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>WhatsThisCursor</cursorShape>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="boton1">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true"> border-style: outset;
border-width: 5px;
border-radius: 10px;
border: 1px black solid;
color: black;
background-color: orange;
padding: 6px;
</string>
              </property>
              <property name="text">
               <string>Ver explicación</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="boton2">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true"> border-style: outset;
border-width: 5px;
border-radius: 10px;
border: 1px black solid;
color: black;
background-color: orange;
padding: 6px;
</string>
              </property>
              <property name="text">
               <string>Ver explicación</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="Boton_Buscar">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">color: white;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
background-image: url(Menu2.png)</string>
           </property>
           <property name="text">
            <string>Buscar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Expl">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">color: white;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
background-image: url(Menu2.png)</string>
           </property>
           <property name="text">
            <string>Modo Experto</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
